Packs/Phishing/Scripts/PhishingDedupPreprocessingRule/PhishingDedupPreprocessingRule.py

```diff
@@ -204,23 +222,28 @@ def main():
     FROM_FIELD = input_args.get('emailFrom', FROM_FIELD)
     FROM_POLICY = input_args.get('fromPolicy', FROM_POLICY)
     existing_incidents = get_existing_incidents(input_args)
+    demisto.results('found {} incidents by query'.format(len(existing_incidents)))
     if len(existing_incidents) == 0:
         create_new_incident()
         return
     new_incident = demisto.incidents()[0]
     new_incident_df = preprocess_incidents_df([new_incident])
+    if len(new_incident_df) == 0:  # len(new_incident_df)==0 means new incident is too short
+        create_new_incident()
+        return
     existing_incidents_df = preprocess_incidents_df(existing_incidents)
```

Comment:
I notice that the preprocess_incidents_df doesn't really support the case when the fields are empty - you should validate the fields exist in the current incident (throw error if not) and filter out incidents  where those field does not exists (for the text fields you can do a condition that at least one of them is not empty maybe)

Reply:
We want to return error if incident has no text fields (if it's run in the phishing investigation playbook)?
 Currently incidents with no text fields will have short length (1) and will be eliminated by their length